MVP/run

```diff
@@ -1,65 +1,71 @@
 #!/bin/bash
 
+# Function to convert Windows path to Unix path
+convert_path() {
+  echo "$1" | sed 's/\\/\//g'
+}
+
 case "$1" in
-  install)
-    echo "Installing dependencies..."
-    npm install
-    exit $?
-    ;;
-  
-  test)
-    echo "Installing dependencies..."
-    npm install
-    
-    echo "Building the project..."
-    npm run build
-    if [ $? -ne 0 ]; then
-      echo "Build failed."
-      exit 1
-    fi
+install)
+  npm install >/dev/null 2>&1
+  npm run build >/dev/null 2>&1 # suppressing output
+  node -e "require('./dist/Output/CLI').printDependencyCount()"
+  exit $?
+  ;;
+
+test)
+  npm install >/dev/null 2>&1
+
+  npm run build >/dev/null 2>&1
+  if [ $? -ne 0 ]; then
+    echo "Build failed."
+    exit 1
+  fi
+
+  mkdir -p coverage
+
+  touch coverage/output.txt
+
+  npm run test >/dev/null 2>&1
+  if [ $? -ne 0 ]; then
+    node -e "require('./dist/Output/CLI').printTestResults()"
```

Comment:
there should be no command needed after npm run test, posttest is ran automatically